servers/jax-rs-tests/src/main/java/org/projectnessie/jaxrs/AbstractRestNamespace.java

```diff
@@ -91,13 +100,21 @@ public void testNamespaces(String namespaceName) throws BaseNessieClientServerEx
   @Test
   public void testNamespacesRetrieval() throws BaseNessieClientServerException {
     Branch branch = createBranch("namespace");
-    Namespace one = Namespace.parse("a.b.c");
-    Namespace two = Namespace.parse("a.b.d");
-    Namespace three = Namespace.parse("x.y.z");
-    Namespace four = Namespace.parse("one.two");
+
+    ThrowingExtractor<String, Namespace, ?> createNamespace =
```

Comment:
why are these changes necessary?

Reply:
to have the content-ID which is part of the .equals() comparison below